src/main.cpp

```diff
@@ -179,80 +179,92 @@ int main(int argc, const char* argv[]) {
     app.add_option("--scratch", scratch_dir, "Directory for intermediate code output")
         ->capture_default_str()
         ->ignore_case();
-    app.add_option("--units", units_dir, "Directory of units lib file")->capture_default_str()->ignore_case();
```

Comment:
this is an unrelated change, is that because you changed clang-format?

Reply:
No idea why. But it is mandatory